### ISSUE TITLE
Nach hiringorganization bei BITE Requests filtern

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/rrze-jobs#59]
Der Shortcode muss erwieztert werden um in dem BITE-Feld "hiringorganization" nach einer FAUORG Nummer zu suchen.

hiringorganization wird bereits von BITE geliefert im Custom Field.

Wir müssen im Shortcode aber noch ein Parameter einbauen, der dann die Ausgaben danach filtert.

Bspw.
[jobs fauorg="nummer" ..]

Wenn die Nummer eine gültige FAU.ORG Nummer ist, dann muss diese mit dem Feldinhalt hiringorganization in 
 BITEs Rückgabe vergleichen weren.
**Achtung: Die von BITE gelieferte hiringOrganization wird zuwischenzeitlich sanitized mit den Werten aus OrgData.php!!**

Daher schauen, dass man auch noch der OrgNummer matcht und nicht nach einem Namen.

**Comment by cassandre** [comment]
B-ITE liefert aber keine FAU.ORG-Nr. zurück. Es gibt nur 
```
[hiringOrganization] => Array
        (
            [name] => Regionales Rechenzentrum Erlangen
            [url] => https://rrze.fau.de
        )
```
und 
```
[_provider-values][hiringorganization_fauorg] => x
```
Wird dieses letztere Feld irgendwann noch mal sinnvoll gefüllt?

**Comment by xwolfde** [comment]
das hab ich doch geschrieben, dass der sanitizer die orgnummer wegmacht..
die orgnummer wird von BITE im custom.field hiringOrganization übergeben.

Das kann man auch über die API-Testcase https://api.b-ite.io/docs/ prüfen. Dort mit der API.-Key Authentifizieren und dann einen Rerquesat in /search oder /getPosting machen

**Comment by cassandre** [comment]
Nein, wird sie nicht. Im Feld hiringorganization wird der Name der Organisation im Klartext übergeben. Die FAU.ORG-Nummer ist nicht im Datensatz von BITE enthalten. Ein Vergleich/Filter ist nur über den Klartext möglich.

![Bildschirmfoto 2023-03-23 um 14 21 42](https://user-images.githubusercontent.com/4435815/227217761-d868fb17-7bb2-400f-9eb7-7f47ff75da2e.png)

**Comment by xwolfde** [comment]
wtf?
<img width="714" alt="grafik" src="https://user-images.githubusercontent.com/1070669/227218938-bb85c638-bd6f-4213-b1d6-0286def78d5a.png">

**Comment by xwolfde** [comment]
magst du es ins main mergen und dann ausrollen?

**Comment by cassandre** [comment]
Done.
